README.md

```diff
@@ -19,10 +19,10 @@ Installs iptables and provides a custom resource for adding and removing iptable
 
 ## Resources
 
-- [iptables_packages](https://github.com/chef-cookbooks/iptables/tree/master/documentation/iptables_packages.md)
-- [iptables_service](https://github.com/chef-cookbooks/iptables/tree/master/documentation/iptables_service.md)
-- [iptables_chain](https://github.com/chef-cookbooks/iptables/tree/master/documentation/iptables_chain.md)
-- [iptables_rule](https://github.com/chef-cookbooks/iptables/tree/master/documentation/iptables_rule.md)
+- [iptables_packages](documentation/iptables_packages.md)
```

Comment:
We should leave the full links as otherwise it does not work in supermarket :(

Reply:
Ah I did wonder why they weren't relative